### PR TITLE
Project - Portfolio Setup and Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by sja-thedude**
- HTML file has been Added
- CSS File has been Added
- Images has been Added
- Readme File has been Updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-Project-Portfolio-setup-and-mobile-version-skeleton
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project : Setup And Mobile Version Skeleton
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```
Comment (SajjadAhmad14) [inline README.md:7]: Please add a description of the project features here. Moreover, please add steps to run the project locally. README should be professional. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+    <header class="header">
+      <h3 class="logo">SJA</h3>
+      <nav class="nav">
+      <img src="img/Union.png" alt="Union"/>
+		</nav>
+	</header>
```
Comment (SajjadAhmad14) [inline index.html:16]: Please wrap the logo in a link tag.
Please use font-family for the header section from the design page.
Please align the logo and hamburger menu vertically. Kindly see the screenshot below for reference.
header

File: index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+  <section class="about">
+          <h1>Laura Jung</h1>
+          <h3>Software Developer</h3>
+          <p>I can help you build a product ,<br> feature or website Look through <br>some of my work and experience!<br> If you like what you see and have a <br>project you need coded, <br> don’t hestiate to contact me.
+          </p>
```
Comment (SajjadAhmad14) [inline index.html:23]: The paragraph (e.g., "I'm a software ...") should have a p tag.
Please use colors and font-family for about section from the design page.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      </nav>
+    </header>
+
+
+  <section class="about">
+          <h1>SJA</h1>
+          <p class="para">Software Developer</p>
+          <p class="par">I can help you build a product ,<br /> feature or website. Look through <br/>some of my work and experience!<br> If you like what you see and have a <br>project you need coded, <br> don’t hestiate to contact me.
+          </p>
+</section>
```
Comment (safafa) [inline index.html:26]: image

- [ ] As the previous reviewer suggested. You should use the same font family and colors used in the design. check the Figma template for style information. Respecting the original design is one of the project's requirements as it's an important skill for a front-end developer. 
original design :
image
You can use this link to lookup [the font](https://fonts.google.com/)

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @sja-thedude , Good job working on some of the changes requested by the previous reviewer 👏🏼 👏🏼 

But,  there are still some which aren't addressed yet.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Mihndim2020 — APPROVED** [review]
Hi @sja-thedude, 

Good job implementing the changes requested by the previews reviewer!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**